src/tracker/psmove_tracker.c

```diff
@@ -414,7 +436,10 @@ PSMoveTracker *psmove_tracker_new() {
         camera = 0;
     }
 #endif
-
+#if defined(_WIN32) && defined(CAMERA_CONTROL_USE_PS3EYE_DRIVER)
+    printf("\nDetected PS3EYEDriver in Windows. You may have to `SET %s=2` (or =1).\n",
```

Comment:
That looks like a message that says "we didn't spend time making sure we find the right camera, so you now have to do this yourself by trial'n'error via environment variables". If `CAMERA_CONTROL_USE_PS3EYE_DRIVER` is used, make sure that it prefers that camera driver over other cameras, and remove this message.